src/models/IDE/VisualStudioCode.py

```diff
@@ -131,3 +131,11 @@ def refresh_configuration(self):
         # Click 'Refresh configurations' button
         self.wait_find_element(locator_type="image", locator="refresh_configurations.png")
         self.click_element(locator_type="image", locator="refresh_configurations.png")
+
+    def open_plugin_info(self, plugin):
```

Comment:
Is this method common for both intelliJ and vscode ?
If yes then we can put it in a common place.